prometheus_client/samples.py

```diff
@@ -32,12 +32,15 @@ def __gt__(self, other):
 
 
 # Timestamp and exemplar are optional.
-# Value can be an int or a float.
+# Value must be a float.
 # Timestamp can be a float containing a unixtime in seconds,
 # a Timestamp object, or None.
 # Exemplar can be an Exemplar object, or None.
-Sample = namedtuple('Sample', ['name', 'labels', 'value', 'timestamp', 'exemplar'])
-Sample.__new__.__defaults__ = (None, None)
+sample = namedtuple('Sample', ['name', 'labels', 'value', 'timestamp', 'exemplar'])
+
+# Wrap the namedtuple to provide eage type-checking that value is a float.
+def Sample(name, labels, value, timestamp = None, exemplar = None):
```

Comment:
Watch your whitespace

Reply:
removed whitespace around equals and added two spaces around top-level members - is this ok?